src/main/java/org/folio/circulation/support/Clients.java

```diff
@@ -39,8 +39,8 @@ private Clients(OkapiHttpClient client, WebContext context) {
       locationsStorageClient = createLocationsStorageClient(client, context);
       materialTypesStorageClient = createMaterialTypesStorageClient(client, context);
       proxiesForClient = createProxyUsersStorageClient(client, context);
-      loanRulesClient = new LoanRulesClient(client, context);
-      loanRulesStorageClient = createLoanRulesStorageClient(client, context);
+      circulationRulesClient = new CirculationRulesClient(client, context);
+      circulationRulesStorageClient = createCirculationRulesStorageClient(client, context);
```

Comment:
@rladdusaw It appears the path used within this client is still `/loan-rules-storage` please can this be updated, along with the fake